package.json

```diff
@@ -12,6 +12,7 @@
     "url": "https://github.com/hostnet/resolver-lib/issues"
   },
   "dependencies": {
+    "brotli": "1.3.2",
```

Comment:
We might consider moving this somewhere else in the future, not sure if NodeJS has a `suggest` equivalent. Bit of a shame to have these dependencies if you are not using them.

I made #63 to have a reminder.

Reply:
well, the first start was moving all the require statements in the processor.js, which this PR already did.....